backend/tests/test_git_ops.py

```
@@ -154,37 +161,49 @@ def test_error_for_missing_repo(self, mock_exists):
+        # Mock resolve to return a path that passes the security check
+        mock_resolved = MagicMock(spec=Path)
+        mock_resolved.is_relative_to.return_value = True
+        mock_resolved.suffix = ".txt"
+        mock_resolved.stat.return_value.st_size = 11
+        mock_resolved.read_text.return_value = "hello world"
+        mock_resolve.return_value = mock_resolved
+        
+        result = read_file("user", "repo", "file.txt")
+        assert result["status"] == "success"
+        assert result["content"] == "hello world"
+        assert result["binary"] == False
```

Ruff (0.14.10)

Avoid equality comparisons to `False`; use `not result["binary"]:` for false checks

Replace with `not result["binary"]`

(E712)